db/seeds.rb

```diff
@@ -1,504 +1,746 @@
 Ingredient.create(name:'Acid Malt', sub_type: 'Grain',
- extract_potential: 1.027, alpha_acid: nil, attenuation: nil, main_class: "Grain")
+  extract_potential: 1.027, alpha_acid: nil,
+  attenuation: nil, main_class: "Grain")
 Ingredient.create(name:'Amber Malt', sub_type: 'Grain',
- extract_potential: 1.035, alpha_acid: nil, attenuation: nil, main_class: "Grain")
+  extract_potential: 1.035, alpha_acid: nil, 
+  attenuation: nil, main_class: "Grain")
 Ingredient.create(name:'Aromatic Malt', sub_type: 'Grain',
- extract_potential: 1.036, alpha_acid: nil, attenuation: nil, main_class: "Grain")
+  extract_potential: 1.036, alpha_acid: nil, 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.